critiquebrainz/frontend/external/bookbrainz_db/edition_group.py

```diff
@@ -0,0 +1,77 @@
+import uuid
+from brainzutils import cache
+from typing import List
+import sqlalchemy
+import critiquebrainz.frontend.external.bookbrainz_db as db 
+from critiquebrainz.frontend.external.bookbrainz_db import DEFAULT_CACHE_EXPIRATION
+from critiquebrainz.frontend.external.bookbrainz_db.identifiers import fetch_identifiers
+from critiquebrainz.frontend.external.bookbrainz_db.relationships import fetch_relationships
+
+
+def get_edition_group_by_bbid(bbid: uuid.UUID) -> dict:
+    """
+    Get info related to an edition group using its BookBrainz ID.
+    Args:
+        bbid : BBID of the edition group.
+    Returns:
+        A dictionary containing the basic information related to the edition group.
+    """
+    edition_group = fetch_multiple_edition_groups([bbid])
+    if not edition_group:
+        return None
+    return edition_group[bbid]
+
+
+def fetch_multiple_edition_groups(bbids: List[uuid.UUID]) -> dict:
+    """
+    Get info related to multiple edition groups using their BookBrainz IDs. 
+    Args:
+        bbids (list): List of BBID of edition groups.
+    Returns:
+        A dictionary containing info of multiple edition groups keyed by their BBID.
+    """
+    if bbids == []:
+        return {}
+
+    key = cache.gen_key('edition-groups', bbids)
```

Comment:
to help us in the future, let's prefix all bookbrainz cache uses with `bb_` or similar (especially the 'identifier' one in identifiers.py, that word is common enough that it could conflict with something else in the future)